src/components/Button/src/Button.vue

```diff
@@ -17,6 +17,19 @@
 		v-bind="$attrs"
 		v-on="$listeners"
 	>
+		<div
+			v-if="variant === 'success' || 'error'"
```

Comment:
Let's make this a method, `isSuccessOrErrorVariant(variant)`

Reply:
or `variantHasPrefixIcon(variant)`